specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/cluster.json

```diff
@@ -1933,6 +1956,148 @@
         }
       }
     },
+    "PrivateLinkConfiguration": {
+      "type": "object",
+      "description": "The private link configuration.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The private link configuration id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of private link configuration."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the private link configuration."
+        },
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkConfigurationProperties",
+          "description": "The private link configuration properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "PrivateLinkConfigurationProperties": {
+      "type": "object",
+      "description": "The private link configuration properties.",
+      "properties": {
+        "groupId": {
```

Comment:
>groupId [](http://example.com/codeflow?start=9&length=7)

The description doesn't make this property sound very ID-like. Is it really an ID, or actually a name, or something else?

Reply:
It is not a resource id, it is more like a name, actually the value is like "gateway","headnode"